e2e/wdio/platform/tests/upload-collection.e2e-spec.ts

```diff
@@ -0,0 +1,164 @@
+    refreshPage, scrollIntoView, setValue,
+    waitForPresent
+} from '../../driver/wdio';
+import {
+    columnHeaderTestArr,
+    testArr, testFolder1,
+    testFolderArr
+} from '../fixtures/appData/upload-collection-content';
+
+describe('Upload collection test suite', function() {
+    const uploadCollectionPage = new UploadCollectionPo();
+    const {
+        defaultExample, disableExample, readonlyExample, turnOffExample, buttons, tableItems, menuItem, tablePages,
+        tableResult, linkNext, linkPrevious, inputFields, columnHeaders, tableContent, transparentButton,
+        dialogInputField, dialogCreateButton, tableItemCount, menuButton
+    } = uploadCollectionPage;
+
+    beforeAll(() => {
+        uploadCollectionPage.open();
+    }, 1);
+
+    afterEach(() => {
+        refreshPage();
+        waitForPresent(defaultExample);
+    }, 1);
+
+
+    it('should check the possibility of creating table item for Default and Without Pagination and Search examples', () => {
+        creatingFolder(defaultExample);
+        creatingFolder(turnOffExample);
+    });
+
+    it('should check the possibility of removing table item for Default and Without Pagination and Search examples', () => {
+        removingItem(defaultExample);
+        removingItem(turnOffExample);
+    });
+
+    it('should check the choice how many items display on page for all examples', () => {
+        checkDisplayedItemsPerPage(defaultExample);
+        checkDisplayedItemsPerPage(disableExample);
+        checkDisplayedItemsPerPage(readonlyExample);
+    });
+
+    it('should check selected pages by clicking each option for all examples', () => {
+        checkSelectedPages(defaultExample);
+        checkSelectedPages(disableExample);
+        checkSelectedPages(readonlyExample);
+    });
+
+    it('should check selected pages by clicking previous and next link for all examples', () => {
+        checkSelectedPagesByNextPrevious(defaultExample);
+        checkSelectedPagesByNextPrevious(disableExample);
+        checkSelectedPagesByNextPrevious(readonlyExample);
+    });
+
+    it('should check placeholders all examples', () => {
+        const inputLength = getElementArrayLength(inputFields);
+        for (let i = 0; i < inputLength; i++) {
+            expect(getElementPlaceholder(inputFields, i)).toBe('Search');
+        }
+    });
+
+    it('should check table column headers displayed correctly for all examples', () => {
+        checkColumnHeaderText(defaultExample);
+        checkColumnHeaderText(disableExample);
+        checkColumnHeaderText(readonlyExample, ' div');
+        checkColumnHeaderText(turnOffExample, ' div');
+    });
+
+    it('should check searching table contents for all examples', () => {
+        checkAbleToSearch(defaultExample);
+        checkAbleToSearch(disableExample);
+        checkAbleToSearch(readonlyExample);
+    });
+
+
+    it('should check orientation', () => {
+        uploadCollectionPage.checkRtlSwitch();
+    });
+
```

Comment:
is there dynamic content preventing visual regression from being added?

Reply:
Yes, tables content changes after each refresh